qsiprep/data/pipelines/pyafq_tractometry.json

```diff
@@ -16,7 +16,7 @@
                 "directions": "prob",
                 "max_angle": 30.0,
                 "sphere": "",
-                "seed_mask": "",
+                "seed_mask": "ImageFile('suffix': 'WM', {'scope': 'qsiprep', 'space': None})",
```

Comment:
Hey @36000 : should this use an `ImageFile` or a `MaskFile` object? Would this work as written here? That is, do these names (`ImageFile`, `MaskFile`) exist in the namespace in which this config is run?

Reply:
Also, can anyone tell me how mrtrix pipelines choose where to place seeds? In particular, the msmt_noACT pipeline?